lm_eval/api/task.py

```diff
@@ -669,9 +669,9 @@ def __init__(
             check_choices = test_target
 
         for choice in check_choices:
-            choice_has_whitespace = True if " " in choice else False
+            choice_has_whitespace = True if choice.startswith(" ") else False
             delimiter_has_whitespace = (
-                True if " " in self._config.target_delimiter else False
+                True if self._config.target_delimiter.startswith(" ") else False
```

Comment:
could be newline. Maybe string[0].isspace()?

Reply:
good catch thanks! should also be string[-1] i believe